src/test/java/org/fundacionjala/trello/pages/menus/MenuBoards.java

```diff
@@ -48,6 +49,7 @@ public boolean isDisplayed() {
     public BoardPage openPersonalBoard(final String boardName) {
         String personalBoard = String.format(PERSONAL_BOARD, boardName);
         WebElement boardElement = driver.findElement(By.xpath(personalBoard));
+        action.waitForPageLoadComplete(DELAY);
         boardElement.click();
```

Comment:
use the action class for this click.

Reply:
This is not my code. I just added that wait because the test was failing